source/custom/xnetsocket_converters.h

```diff
@@ -131,23 +192,41 @@ struct TimeValHolder {
 };
 
 template <typename TSockAddr>
-inline SockAddrHolder convert_from_grpc(const SockAddr& input)
+inline SockAddrInputConverter convert_from_grpc(const SockAddr& input)
 {
-  return SockAddrHolder(input);
+  return SockAddrInputConverter(input);
 }
 
-template <typename TSet>
-inline SetHolder convert_from_grpc(
+inline void convert_to_grpc(const SockAddrOutputConverter& input, SockAddr* output)
```

Comment:
nit: I think `input` for the SockAddrOutputConverter param is a little confusing. just because there's a lot of input/output. Maybe `stored`